src/pseudopeople/interface.py

```diff
@@ -931,3 +1142,24 @@ def get_dataset_filepaths(source: Path, dataset_name: str) -> list[Path]:
     dataset_paths = [x for x in directory.glob(f"{dataset_name}*")]
     sorted_dataset_paths = sorted(dataset_paths)
     return sorted_dataset_paths
+
+
+def set_up_dask_client() -> None:
+    """Sets up a Dask client if one is not already running."""
+    from dask.distributed import get_client
+
+    # Determine whether or not a Dask client is already running. If not,
+    # create a new one.
+    try:
+        get_client()
+    except ValueError:
+        # No Dask client is running so we create one.
+        from dask.distributed import LocalCluster
+        from dask.system import CPU_COUNT
+
+        # extract the memory limit from the environment variable
+        cluster = LocalCluster(  # type: ignore [no-untyped-call]
+            n_workers=CPU_COUNT,
+            threads_per_worker=1,
+        )
+        cluster.get_client()  # type: ignore [no-untyped-call]
```

Comment:
I have a vague memory of needing to have this client object in scope for it to be used. But I presume your testing here has ensured this client is used? I wonder if `.get_client` is even necessary?

Reply:
Oh, I should probably add to the test that the client is actually type LocalCluster. Then I think I'd be happy b/c that's def not just the default

Reply:
Shoot, the dask-default client happens to be a LocalCluster as well. I'll add a name to our new default (name = "default" unless you disagree) and assert that